content/en/1.index.md

```diff
@@ -1,44 +1,246 @@
 ---
 title: Introduction
+hideToc: true
+layout:
+  aside: false
+  fluid: true
+bulletLinks: ['bulletLinks', 'bulletLinks']
```

Comment:
Can you please fix this?

Reply:
@vinayakkulkarni : Made changes under this commit - https://github.com/geospoc/gqi-mapper-documentation/pull/81/commits/0a8d40df9cfe717421205cbde015ba35df09e0c0 & pushed.

Also, incorporated changes as discussion happened with @puneethshankar 
Consider review & merge of this PR, Thanks....